### DISCUSSION TITLE
Can anyone help me with this please? while installing brotlipy

### DISCUSSION THREAD

**Discussion by Bharath23453** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#306] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/60808613/128004600-740d4299-f85c-4710-9bae-f1ab45552126.png)

**Comment by fakeid30** [comment, marked as answer]
If it's still an issue you need to install Visual C++ 14. Check our dev branch readme.
